README.md

```diff
@@ -431,6 +436,10 @@ content
 -------
 Content of the PAM file for the service
 
+lines
```

Comment:
Neither the lines or content parameters are required, not setting either will result in an empty service file... which is replicating the behavior of not setting the content parameter before this patch.